wordpress-mobile/WordPressCom-Stats-iOS
# Tapping views/visitors/likes/comments in Insights switches to Days summary type

Closes #261 

When tapping the views/visitors/likes/comments buttons in Insights, the view is now switched to the graph, selects Days and then changes the summary type to the appropriate one according to what tapped.

## What to Test
1. Selection of each item in Insights. Make sure it switches to Days and selects the right summary type.
2. Switch back and forth between periods and selected bars in the graph, do #1 again and make sure it works.
3. Try to break it.

## How to Test
Use StatsDemo built today (18-JUN) - it includes this PR and the other two dependent PRs still open.

Needs Review: @jleandroperez, @isaackeyet
